### ISSUE TITLE
Can't load extension.

### ISSUE THREAD

**Issue by nanashi89** [DarkoPendragon/minds-darker#1]
"Go to Settings > Extensions > Manage Extensions
Enable Developer Mode
Download the latest release under 'Releases' to the right and unzip it to desired location
Click Load unpacked extension
Select the folder you just made with the manifest.json in it"

"manifest.json" does not appear in the "select the extension directory" when I try to load it.

I'm on the latest build of the Brave web browser.

**Comment by DarkoPendragon** [comment]
Yes, sorry about that. I reference a file that isn't used anymore. That's my fault. I'm just tidying up and update and that will be fixed.

Edit: forgot to mention, if you replace the `manifest.json` with this it'll fix it.
```json
{
  "manifest_version": 2,
  "name": "Minds Darker",
  "author": "Darko Pendragon",
  "description": "Attempts to make Minds.com's dark mode better",
  "version": "1.2.1",
  "icons": {
    "16": "./assets/icon.png",
    "48": "./assets/icon.png",
    "128": "./assets/icon.png"
  },
  "permissions": [
    "storage"
  ],
  "content_scripts": [
    {
      "matches": [
        "https://www.minds.com/*"
      ],
      "js": ["./js/jquery.min.js","./js/jquery.initialize.min.js","./js/content.js"],
      "css": ["./assets/root.css"]
    }
  ],
  "background": {
    "matches": [
      "https://www.minds.com/*"
    ]
  },
  "options_page": "./popup/popup.html",
  "browser_action": {
    "default_title": "Minds Darker",
    "default_icon": "./assets/icon.png",
    "default_popup": "./popup/popup.html"
  }
}
```

**Comment by DarkoPendragon** [comment]
Fixed as of 1.2.1
